### PR TITLE
Release 0.1.0

### PR DESCRIPTION
**Pull request by slifty**
This PR makes the changes needed for our 0.1.0 release.

Next steps will be to create a release via GitHub's UX (which should, in theory, then also publish to npm automatically)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,9 +7,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-## [0.0.0] - 2022-03-21
+## [0.1.0] - 2022-03-21
```
Comment (cecilia-donnelly) [inline CHANGELOG.md:10]: Should this be today's date?

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,9 +7,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 ### Added
-- A placeholder project / release with no completed functionality.
+- The `getArchives` function, which will get a list of archives associated with the authorized account.
+- The `Archive` type.
+- The `ClientConfig` type which allows an sdk user to specify their credentials and other configuration settings.
 
-[Unreleased]: https://github.com/permanentorg/permanent-sdk/base/compare/@tvkitchen/base-constants@v0.0.0...HEAD
-[0.0.0]: https://github.com/permanentorg/permanent-sdk/base/releases/tag/v0.0.0
+[Unreleased]: https://github.com/permanentorg/permanent-sdk/base/compare/@tvkitchen/base-constants@v0.1.0...HEAD
```
Comment (cecilia-donnelly) [inline CHANGELOG.md:16]: What's `@tvkitchen` here?  I know that's another package you own, @slifty, but are we using it for something in the SDK? (Sorry if I'm behind the times!)

**Review by cecilia-donnelly — COMMENTED** [review]
Pretty exciting!

File: CHANGELOG.md (hunk 2 above)
Comment (slifty) [reply CHANGELOG.md:16]: Lol I am so glad I asked for a review.

All the things just blurred together.

File: CHANGELOG.md (hunk 1 above)
Comment (slifty) [reply CHANGELOG.md:10]: Yes!

**Review by cecilia-donnelly — APPROVED** [review]
yey!

**Pull request merged**